modules/EsetTools.py

```
@@ -132,10 +146,10 @@ def sendRequestForVPNCodes(self):
         console_log('\nSending a request for VPN subscriptions...', INFO)
         self.driver.get("https://home.eset.com/security-features")
         try:
-            uCE(self.driver, f'return {GET_EBAV}("button", "data-label", "security-feature-tile-1-button") != null', max_iter=10)
-            uCE(self.driver, f'return {CLICK_WITH_BOOL}({GET_EBAV}("button", "data-label", "security-feature-tile-1-button").querySelector(".css-l83yl9"))', max_iter=5)
+            uCE(self.driver, f'return {CLICK_WITH_BOOL}({GET_EBAV}("button", "data-label", "security-feature-explore-button"))', max_iter=10)
         except:
```

Ruff

Do not use bare `except`

(E722)